api/LMPlatform.Models/Concept.cs

```diff
@@ -12,6 +12,51 @@ public class ConceptQuestions: ModelBase
         public virtual Concept Concept { get; set; }
         public virtual Question Question { get; set; }
     }
+    public class ConceptMonitoring : ModelBase
```

Comment:
Need create a separate file for each class

Reply:
https://github.com/hawkrai/CATSdesigner/pull/668/commits/612402fff3a8002ff5117220c28ca3080da97cd0